code/game/objects/effects/overlays.dm

```diff
@@ -45,10 +45,10 @@
 /obj/effect/overlay/temp/dir_setting
 	randomdir = FALSE
 
-/obj/effect/overlay/temp/dir_setting/New(loc, set_dir)
+/obj/effect/overlay/temp/dir_setting/Initialize(mapload, set_dir)
 	if(set_dir)
 		setDir(set_dir)
-	..()
```

Comment:
This was running code before parent New() that would now happen after New().

Does this matter in this case?